tutorials/graph-hello-world/graph-hello-world.md

```diff
@@ -0,0 +1,166 @@
+
+[DONE]
+[ACCORDION-END]
+
+[ACCORDION-BEGIN [Step 4: ](Prepare graph.js)]
+
+First, we will create a boilerplate file in the `src` folder, called `graph.js`. This file will contain a class called `Graph`, which provides a nice wrapper for using SAP Graph. At this time, we will show how to read data, via a `get()` function.
+
+Paste the following code into the file and save it. As you can see, it is very simple, and takes advantage of the node-fetch package.
+
+```
+const fetch = require("node-fetch");
+class Graph {
+    async get(req, entity, params) {
+        const url = `https://sandbox.api.sap.com/sapgraph/${entity}${params ? `?${params}` : ""}`;
+        console.log(url) //for debugging
+        const options = {
+            method: "get",
+            headers:{
+                "Accept": "application/json",
+                "apiKey": "your APIkey"
+            }
+        };
+        const response = await fetch(url, options);
+        console.log(`${response.status} (${response.statusText})`) // for debugging
+        const json = await response.json();
+        return json;
+    }
+}
+
+module.exports = Graph;
+
+```
+Note that we hard-coded the SAP Graph server that uses SAP API Business Hub sandbox. This allows us to focus here on the data access API, without requiring you to first configure your own SAP Graph business data graph and deal with all the complex aspects of security and authentication, which will be the subject of future tutorial parts.
+
+Since we are accessing the sandbox landscape data via SAP API Business Hub, you will need to insert your API Key (a short string) into the code above. Where do you get this key? Go and login to <https://api.sap.com/settings> and click on **Show API Key** to see and save it.
+
+[DONE]
+[ACCORDION-END]
+
+[ACCORDION-BEGIN [Step 5: ](Write a Simple Application)]
+
+Now we are ready to write our first, simple server-side application that uses SAP Graph. Paste the following into a new file, called `hellograph.js`, in the `src` folder:
+
+```
+const express = require("express");
+const app = express();
+const Graph = require("./graph");
+const port = 3004;
+
+const graph = new Graph();
+
+app.get('/sap*', async (req, res) => {
+    const response = await graph.get(req, `${req.url}`, "");
+    res.send(`<pre><code>${JSON.stringify(response, null, 2)}</code></pre>`);
+});
+
+app.listen(port, () => { console.log(`Explore SAP Graph at http://localhost:${port}`) });
+
+```
+As mentioned, our code uses a popular Node.js package called **Express**. On line 8, we define the action taken when our server-side app is called from a browser. Here, we simply take the URL that we received (`req.url`), and pass it through, as-is, to SAP Graph. We then show the returned data from SAP Graph as a raw result in the browser screen.
+
+Our server-side app is ready. To run it, use your terminal console to enter:
```

Comment:
Use You and not the royal We.(unless, in rare cases, you are referring to SAP).

Reply:
Rephrased.